src/MusicalScore/Graphical/MusicSystemBuilder.ts

```diff
@@ -134,7 +134,9 @@ export class MusicSystemBuilder {
             }
             const measureFitsInSystem: boolean = this.currentSystemParams.currentWidth + totalMeasureWidth + nextMeasureBeginInstructionWidth < systemMaxWidth;
             const doXmlPageBreak: boolean = this.rules.NewPageAtXMLNewPageAttribute && sourceMeasure.printNewPageXml;
-            const doXmlLineBreak: boolean = doXmlPageBreak || // also create new system if doing page break
+            const impliedSystemBreak: boolean = doXmlPageBreak || // also create new system if doing page break
```

Comment:
an implied system break is not a page break.
this variable should be:
```typescript
const impliedSystemBreak: boolean = sourceMeasure.printNewPageXml && this.rules.PageBreakImpliesSystemBreak;
```
as I wrote in my other comment. If we have a page break, we don't need a system/line break.

Reply:
Wait, ignore the previous comment, this was as before and necessary. Sorry, I got confused there.

Reply:
Actually this is fine anyways, because logically, a page break also implies a system break anyways.